### PR TITLE
Add advanced image options

### PR DESCRIPTION
**Pull request by Zrgisa**
Image cropping within the Gutenberg block for the core image block and core cover block.

### CODE REVIEW

File: src/extensions/advanced-image/index.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
+import AttachmentCropControl from '../../components/image-crop-control/attachment-crop-control';
+
+import { assign } from 'lodash';
```
Comment (paranoia1906) [inline src/extensions/advanced-image/index.js]: I changed this to import Syntax so ESLint was not not warning.

File: src/extensions/advanced-image/index.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+		// Only Gallery images supported at this time
+		if ( ! currentAttributes.id ) {
+			return (
+				<BlockEdit { ...props } />
+			);
+		}
+
+		const updateImage = function() {
+			jQuery.post( global.ajaxurl, {
```
Comment (paranoia1906) [inline src/extensions/advanced-image/index.js]: Changed `ajaxurl` to `global.ajaxurl`. This is to avoid ESLint warnings.

**Comment by paranoia1906** [comment]
Tested and seems to be working well. Made a few changes to minimize ESLint warnings.

Functionality is enabled only for `core/cover` and `core/image` blocks. I tried enabled for `coblocks/gallery-masonry` and other gallery blocks but it seems that this extension may need further modification before it may be used by CoBlocks Blocks.

**Comment by richtabor** [comment]
Current flow: ScreenFlow

File: src/extensions/advanced-image/index.js (hunk 2 above)
Comment (EvanHerman) [reply src/extensions/advanced-image/index.js]: For future reference, we can also throw `/* global ajaxurl */` at the top of the file to mark it as global in ESLint.

**Comment by richtabor** [comment]
@Zrgisa I tweaked the loading a bit to be a bit more Gutenberg-like. It's _ok_ for now, but do you think it's possible to employ the core `Spinner` component when it's loading?

Here's a screenshot:
sss

**Pull request merged**